client.py

```
@@ -146,25 +149,71 @@ async def handle_add_game_request(message, game_name):
         reply = 'This game doesn\'t exist on steam, try gamepass.'
     await message.reply (reply)
 
+async def handle_delete_game_request_from_match(message, index_string):
+    index = int(index_string) - 1
+    game_matches = client_state.get_game_matches()
+    if len(game_matches) == 0:
+        print('game matches is empty')
+        return
+    game_tupple = game_matches[index]
+    deleted_game_result = delete_game(game_tupple)
+    print('Deleted game result is:', deleted_game_result)
+    if deleted_game_result == True:
+        reply = f'**{game_tupple[1]}** was successfully deleted from your wishlist.'
+    await message.reply(reply)

```

We are conditionally sending a reply? So if the game doesn't get deleted, it is a silent failure?

---

Yes, currently a silent failure, shall fix in next commit